### ISSUE TITLE
Schedule jobs seem not working.

### ISSUE THREAD

**Issue by Siubaak** [Countly/countly-server#313]
I am working on plugin developing for Countly, and there are some schedule jobs for my plugin 'fund'. But they can't work properly.

I have registered the event '/master' in api.js like below.
```js
plugins.register('/master', function(ob){
    setTimeout(() => {
        require('../../../api/parts/jobs').job('fund:count').replace().schedule('at 1:00 pm');
    }, 11000);
});
```

And I also overwrote the 'run' method in my job 'count.js' like below.
```js
class CountJob extends job.Job {
    run (db, done) {
        doing_something();
        done();
    }
}
```
But I found it never run, no matter how I schedule it like now() or in(5).

Actually I also found that the default jobs like api:clear and api:clearToken didn't run once. Could you please tell me if I did something wrong? I think some documentations might need to be completed about jobs if possible. :)

**Comment by iartem** [comment]
A couple of questions:

1. Where that `count.js` is located? 
2. Is fund plugin enabled?

**Comment by Siubaak** [comment]
1. count.js is located in /fund/api/jobs.
2. I've added it in the plugins.json and it was enable.

I'm sure that my plugins have been working 'cause it's other features worked normally except the schedule job. And I also found that the job scanner had scanned my job and did saving or replacing, but the job seemed not to start following the schedule. Does it delays? Or some settings are missing?

**Comment by iartem** [comment]
You mean our `later` dependency? I don't think we have ever found any problems with it. But in case it misbehaves I'd like to know details.

**Comment by Siubaak** [comment]
Sorry for reply late. Actually later works perfectly. The "later" I downloaded was from a private mirror repo of npm instead of the official npm repo, because it's too slow to download from official repo. But it seemed lack of several important files like . I thought the private mirrorr sever had some problem when synchronizing the official repo. I changed to download all the dependencies from npm finally. So, never mind.
